source/agora/test/ValidatorRecurringEnrollment.d

```diff
@@ -63,3 +63,114 @@ unittest
     // Check if all validators in genesis are enrolled again
     assert(blocks[blocks.length - 1].header.enrollments.length == blocks[0].header.enrollments.length);
 }
+
+// Recurring enrollment with wrong `random_seed`
```

Comment:
A more detailed description needed.
Refer to other test code, please.
https://github.com/bpfkorea/agora/blob/a92aead454b8e93a79ef7c9fccec7f1d4aaacabe/source/agora/test/ValidatorCleanRestart.d#L116-L120